multiplayer/examples/tic-tac-toe/logic.js

```diff
@@ -47,7 +48,31 @@ Rune.initLogic({
       game.lastPlayerId = playerId
 
       if (gameOver) {
-        Rune.gameOver()
+        game.gameOver = true
+
+        if (game.winCombo) {
+          // Last person to make a move is the winner
+          const winner = game.lastPlayerId
+          // The other one must be the loser
+          const loser = game.players.find((id) => id !== winner)
+
+          Rune.gameOver({
+            players: {
+              [winner]: "WON",
+              [loser]: "LOST",
+            },
+            delayPopUp: true,
+          })
+        } else {
+          // Game is a draw
+          Rune.gameOver({
```

Comment:
Interesting, I guess you can't just do a generic game over if you also want to use `delayPopUp: true`

Reply:
It's possible to do, but I thought it would actually make sense to have "everybody lost" as the draw result